packages/grafana-ui/src/themes/GlobalStyles/page.ts

```diff
@@ -7,13 +7,18 @@ export function getPageStyles(theme: GrafanaTheme2) {
 
   return css`
     .grafana-app {
+      position: relative;
       display: flex;
-      align-items: stretch;
-      position: absolute;
-      width: 100%;
+      flex-direction: column;
+      flex-grow: 1;
```

Comment:
I think we can remove this as `flex: 1 1 0;` on line 15 will override it.

Reply:
+1